doc/en/user/source/data/webadmin/layergroups.rst

```diff
@@ -158,3 +158,12 @@ Finally, it is possible to override the service settings and force a **Yes** to
 .. figure:: img/data_layers_root_in_capabilities.png
   
    Layer groups root layer in capabilities options
+
+HTTP Settings
+^^^^^^^^^^^^^
+
+Cache parameters that apply to the HTTP response from client requests.
```

Comment:
Best to clearly specificy what is going to happen if both the group and the layers/grous nested in this group have caching headers setup. I believe the group headers should take precedence, since the request is asking for the group directly, but open to discussion.

Reply:
That was the discussion we had last week. Both could happen. Considering the other comments I think it should be using the layer group HTTP timeout and if not defined it should use layer HTTP timeout, with the exception of remote layers as it already was in the code.

Reply:
A private discussion between us is no good to users. Please document it :-D